kodilla-patterns/src/main/java/com/kodilla/patterns/factory/tasks/DrivingTask.java

```diff
@@ -0,0 +1,50 @@
+package com.kodilla.patterns.factory.tasks;
+
+public class DrivingTask implements Task
```

Comment:
jeśli to miał być immutable - brakuje final przy klasie

Reply:
Poprawiłem